### PR TITLE
Nominate developers in embulk-output-jdbc/mysql/postgresql/redshift/sqlserver

### PR DESCRIPTION
**Pull request by dmikurube**
We're going to release JARs of `embulk-output-jdbc/...` to Maven Central from 0.10.0 (#294).

Maven Central needs the `<developers>` section available in `pom.xml`. In this chance, trying to nominate developers who have contributed a important portion in `embulk-input-jdbc/mysql/postgresql/redshift` from the Git log.

### CODE REVIEW

File: embulk-output-jdbc/build.gradle [hunk 1]
```diff
@@ -6,3 +6,42 @@ embulkPlugin {
...
+                        email = "frsyuki@gmail.com"
+                    }
+                    developer {
+                        name = "Hitoshi Tanaka"
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "takumakanari"
+                        email = "chemtrails.t@gmail.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-jdbc/build.gradle:26]: @takumakanari Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-jdbc/mysql/postgresql` on Maven Central, from your work in #17 and #33. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-jdbc/build.gradle [hunk 2]
```diff
@@ -6,3 +6,42 @@ embulkPlugin {
...
+                    developer {
+                        name = "Tomohiro Hashidate"
+                        email = "kakyoin.hierophant@gmail.com"
+                    }
+                    developer {
+                        name = "Megumi Tomita"
+                        email = "tomykaira@gmail.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-jdbc/build.gradle:34]: @tomykaira Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver` on Maven Central, from your work in #115. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-redshift/build.gradle [hunk 3]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Hiroyuki Sato"
+                        email = "hiroysato@gmail.com"
+                    }
+                    // developer {  // https://github.com/embulk/embulk-output-jdbc/commit/97f3a4ba14ecb99b9bdfa03831a84cb7e1a8e970
+                    //     name = "Lance Norskog"
+                    //     email = "lance@edmodo.com"
+                    // }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:55]: @hito4t Do you remember this merge commit 97f3a4ba14ecb99b9bdfa03831a84cb7e1a8e970 ?

It looks to be commits from Lance Norskog, but not via a pull request.

File: embulk-output-redshift/build.gradle [hunk 4]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                    developer {
+                        name = "Roland Rifandi Utama"
+                        email = "roland_hawk@yahoo.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:59]: @rolandhawk Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #105. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-redshift/build.gradle [hunk 5]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                    developer {
+                        name = "Megumi Tomita"
+                        email = "tomykaira@gmail.com"
+                    }
+                    developer {
+                        name = "Antoine Augusti"
+                        email = "antoine@drivy.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:67]: @AntoineAugusti Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #117 and #119. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-redshift/build.gradle [hunk 6]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                    // developer {  // https://github.com/embulk/embulk-output-jdbc/pull/142
+                    //     name = "Vincent Martinez"
+                    //     email = ""
+                    // }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:71]: @Olaktal Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #142. What do you think? 

I picked up your name from the pull-request and your GitHub user page, but I was not able to find your email. (So, this part is commented out.) Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-redshift/build.gradle [hunk 7]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                    developer {
+                        name = "Toyama Hiroshi"
+                        email = "toyama0919@gmail.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:75]: @toyama0919 Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #114. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-redshift/build.gradle [hunk 8]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                    developer {
+                        name = "Michael Jalkio"
+                        email = "mjalkio@classy.org"
+                    }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:79]: @mjalkio Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #194 and #206. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-redshift/build.gradle [hunk 9]
```diff
@@ -31,3 +31,62 @@ embulkPlugin {
...
+                    developer {
+                        name = "Yutaka Nishimura"
+                        email = "ytk.nishimura@gmail.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-redshift/build.gradle:83]: @ynishi Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-redshift` on Maven Central, from your work in #261. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-sqlserver/build.gradle [hunk 10]
```diff
@@ -26,3 +26,38 @@ embulkPlugin {
...
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "Megumi Tomita"
+                        email = "tomykaira@gmail.com"
+                    }
+                    developer {
+                        name = "uu59"
+                        email = "k@uu59.org"
+                    }
```
Comment (dmikurube) [inline embulk-output-sqlserver/build.gradle:46]: @uu59 Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-sqlserver` on Maven Central, from your work in #121. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-sqlserver/build.gradle [hunk 11]
```diff
@@ -26,3 +26,38 @@ embulkPlugin {
...
+                    developer {
+                        name = "kitsuy"
+                        email = "kitsuy@eac.ne.jp"
+                    }
```
Comment (dmikurube) [inline embulk-output-sqlserver/build.gradle:50]: @kitsuyui Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed a important portion in `embulk-output-jdbc/mysql/postgresql/redshift/sqlserver`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-sqlserver` on Maven Central, from your work in #209 and #214. What do you think? 

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

File: embulk-output-jdbc/build.gradle [hunk 12]
```diff
@@ -6,3 +6,42 @@ embulkPlugin {
...
+                        email = "thitoshi@cac.co.jp"
+                    }
+                    developer {
+                        name = "takumakanari"
+                        email = "chemtrails.t@gmail.com"
+                    }
+                    developer {
+                        name = "Tomohiro Hashidate"
+                        email = "kakyoin.hierophant@gmail.com"
+                    }
```
Comment (dmikurube) [inline embulk-output-jdbc/build.gradle:30]: @joker1007 Hi! We're going to nominate you here, too, as the same as https://github.com/embulk/embulk-input-jdbc/pull/210#discussion_r639415928.

File: embulk-output-redshift/build.gradle (hunk 5 above)
Comment (AntoineAugusti) [reply embulk-output-redshift/build.gradle:67]: Thanks! I'm okay with that. You can use my email hi@antoine-augusti.fr

File: embulk-output-redshift/build.gradle (hunk 8 above)
Comment (mjalkio) [reply embulk-output-redshift/build.gradle:79]: I'm okay with being included, thanks for reaching out! Can you please use `mjalkio@gmail.com` as my email address?

File: embulk-output-jdbc/build.gradle (hunk 12 above)
Comment (joker1007) [reply embulk-output-jdbc/build.gradle:30]: Thanks!!
It's no problem.

File: embulk-output-sqlserver/build.gradle (hunk 11 above)
Comment (kitsuyui) [reply embulk-output-sqlserver/build.gradle:50]: Thanks!
Would you use my name `Yui Kitsu` and email `kitsuyui@kitsuyui.com` ?

File: embulk-output-jdbc/build.gradle (hunk 1 above)
Comment (takumakanari) [reply embulk-output-jdbc/build.gradle:26]: No problem. 
Thanks for your message 👍

**Review by vietnguyen-td — APPROVED** [review]
LGTM

File: embulk-output-redshift/build.gradle (hunk 9 above)
Comment (ynishi) [reply embulk-output-redshift/build.gradle:83]: Hi, Thank you! It is Okay!

File: embulk-output-redshift/build.gradle (hunk 3 above)
Comment (hito4t) [reply embulk-output-redshift/build.gradle:55]: @dmikurube 
I was not able to merge the PR #86 as it was, so I modified and merged.

**Review by hito4t — APPROVED** [review]

File: embulk-output-redshift/build.gradle (hunk 3 above)
Comment (dmikurube) [reply embulk-output-redshift/build.gradle:55]: Thanks! Got it. And the person is no longer on GitHub... Let's skip them.

**Comment by dmikurube** [comment]
Thanks all! Added f9339895194deec3cbc7d432a4289993aff7acec to update contacts based on responses, and too remove nominations of those who had no responses.

I'm going to merge this PR with "Squash and merge".

**Pull request merged**